core/src/main/resources/error/error-classes.json

```diff
@@ -346,6 +346,18 @@
       }
     }
   },
+  "INDEX_ALREADY_EXISTS" : {
```

Comment:
Where is it used?

Reply:
It should appear somewhere in the source code:
```
$ find . -type f -print0|xargs -0 grep 'INDEX_ALREADY_EXISTS'
./core/target/scala-2.12/classes/error/error-classes.json:  "INDEX_ALREADY_EXISTS" : {
./core/src/main/resources/error/error-classes.json:  "INDEX_ALREADY_EXISTS" : {
./sql/core/src/main/scala/org/apache/spark/sql/jdbc/H2Dialect.scala:          // INDEX_ALREADY_EXISTS_1
```
but it is not in fact.

Reply:
Maybe pass the error class here?
https://github.com/apache/spark/blob/288bdd28635d71623dfc6fe82dc1e3aab5fa6c63/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/AlreadyExistException.scala#L84-L85

Reply:
Done, I wonder if this got lost in a merge...